docs: release notes

kpodsiad opened the pull request:
- [ ] add paragraph about bloop jvm settings
- [ ] count closed issues and new features
- [ ] double check if tl;dr doesn't omit any interesting PR
- [ ] pretty much all of `releasing.md` :c

tanishiking commented on website/blog/2022-04-21-aluminium.md:
```diff
@@ -0,0 +1,496 @@
+## Create companion object code action
+
+Thanks to the efforts of [zmerr](https://github.com/zmerr), Metals offers a new code action - `Create companion object`.
+As its name suggests, it can be used to to automatically create a companion object for the given class, trait or enum which
```
Can we drop "which encloses cursor" part? This is an internal implementation detail, and it might be a bit confusing for end uaes.

kpodsiad replied:
I wanted to give some clue when this code action is available. Do you think that GIF is enough?

tanishiking commented on website/blog/2022-04-25-aluminium.md:
```diff
@@ -0,0 +1,498 @@
+- Show better unnaply signatures
+- Provide an easier way to configure bloop settings
+- Better MUnit support in Test Explorer
+
```
```suggestion
- Better sbt BSP integration
- Improved Scala3 support

```

tanishiking reviewed (commented):
@kpodsiad Thank you very much for kicking the release off! I added my part of the contributions.

tgodzik commented on website/blog/2022-04-25-aluminium.md:
```diff
@@ -0,0 +1,516 @@
+
+Thanks to the joined effort of [\#3772](https://github.com/scalameta/metals/pull/3772), [\#3763](https://github.com/scalameta/metals/pull/3763) and [\#3710](https://github.com/scalameta/metals/pull/3710) the Doctor view currently:
+
+- shows Metals server version and information about Java used to run Metals server
```
```suggestion
- shows the Metals server version and information about Java used to run the Metals server
```

tgodzik commented on website/blog/2022-04-25-aluminium.md:
```diff
@@ -0,0 +1,516 @@
+Thanks to the joined effort of [\#3772](https://github.com/scalameta/metals/pull/3772), [\#3763](https://github.com/scalameta/metals/pull/3763) and [\#3710](https://github.com/scalameta/metals/pull/3710) the Doctor view currently:
+
+- shows Metals server version and information about Java used to run Metals server
+- displays compilation status of project
```
```suggestion
- displays the compilation status of project
```

tgodzik commented on website/blog/2022-04-25-aluminium.md:
```diff
@@ -0,0 +1,516 @@
+- Metals has to compile your project and create a debug session
+- debugger has to be initialized (configuration, breakpoints, etc.)
+
+Very often each of these steps needs even a few seconds in order to complete.
```
```suggestion
Very often each of these steps takes up to a few seconds in order to complete.
```

tgodzik commented on website/blog/2022-04-25-aluminium.md:
```diff
@@ -0,0 +1,516 @@
+![run-tests](https://imgur.com/BtbOubC.gif)
+Theme: [One Dark Pro](https://marketplace.visualstudio.com/items?itemName=zhuangtongfa.Material-theme)
+
+It is also possible to just press `F5` in file and Metals will execute main class or run tests in the current file.
```
```suggestion
It is also possible to just press `F5` in file and Metals will execute main class or run tests in the current file if no run configuration is defined.
```

tgodzik commented on website/blog/2022-04-25-aluminium.md:
```diff
@@ -0,0 +1,516 @@
+## Better sbt BSP integration
+
+Metals now doesn't run compilation when sbt generates files under the `src-managed` directory.
+This improvement fixed the continuous compilation when using sbt BSP and some sbt-plugins generate files on compile.
```
```suggestion
The previous behaviour was related to some of the source generating sbt plugins. When used with sbt BSP users would experience  continuous compilation.
```

tgodzik reviewed (changes requested):
Thanks for doing the release!

kpodsiad commented:
@tgodzik @dos65 @tanishiking I've updated release notes with the latest changes + and I've updated the server version. Take a look before I'll push the tag to the github.
